src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -160,11 +182,16 @@ pub fn build_ingest(mut cmd: BuiltinCommand) -> Result<IngestAction, String> {
         },
         "protobuf" => {
             let descriptor_file = cmd.args.string("descriptor-file")?;
+            // This was introduced after the avro format's confluent-wire-format, so it defaults to
+            // false
             let message = cmd.args.string("message")?;
             validate_protobuf_message_name(&message)?;
             Format::Protobuf {
                 descriptor_file,
                 message,
+                confluent_wire_format: cmd.args.opt_bool("confluent-wire-format")?.unwrap_or(false),
+                schema_id: cmd.args.opt_parse::<i32>("schema-id")?.unwrap_or(885),
```

Comment:
885 is an extremely weird default! See below, but I think this should default to `None`, and if it is `None`, then the _current_ schema ID should be looked up (the way it is for Avro data). If it's `Some`, then it's used directly. That'll let you omit it and get the correct schema ID even for non-fresh CSR, and to test the case of a bad schema ID, you can just plug in some impossibly high schema ID like i32::MAX.

Reply:
ah I see. This is actually causing ci to fail I believe, so I need to do that. the hardcoded only works locally, so ill have to play with this